CONTRIBUTING.md

```diff
@@ -34,6 +35,46 @@ go get -u github.com/golang/dep/cmd/dep
 - [OpenPitrix](https://github.com/openpitrix/openpitrix): Application management platform on multi-cloud environment, it provides application template and application management for KubeSphere currently.
 - [SonarQube](https://github.com/SonarSource/sonarqube): Integrated in KubeSphere DevOps, it provides the capability to not only show health of an application but also to highlight issues newly introduced. 
 
+## Building KubeSphere on a local OS/shell environment
+
+### For Quick Taste Binary
+
+```bash
+mkdir ks-tmp
+cd ks-tmp
+echo 'module kubesphere' > go.mod
+echo 'replace (
+      	github.com/kiali/kiali => github.com/kubesphere/kiali v0.15.1-0.20190407071308-6b5b818211c3
+      	github.com/kubernetes-sigs/application => github.com/kubesphere/application v0.0.0-20190518133311-b9d9eb0b5cf7
+      )' >> go.mod
+      
+GO111MODULE=on go get kubesphere.io/kubesphere@cda4f4f05ae1d5c653e7158b8de0164f92341e5c
```

Comment:
@runzexia  I got errors when executed these command by following this guide:

go: k8s.io/apiextensions-apiserver@v0.0.0-20181213153335-0fe22c71c476: unrecognized import path "k8s.io/apiextensions-apiserver" (https fetch: Get https://k8s.io/apiextensions-apiserver?go-get=1: dial tcp 35.201.71.162:443: i/o timeout)
go: error loading module requirements

Reply:
Try it with goproxy  https://github.com/golang/go/wiki/Modules#are-there-always-on-module-repositories-and-enterprise-proxies